src/main/java/com/microservice/file/filemetadate/entity/FileAnalyzeResponse.java

```diff
@@ -0,0 +1,47 @@
+package com.microservice.file.filemetadate.entity;
+
+public class FileAnalyzeResponse {
+    private String fileName;
+
+    private String fileType;
+    private long fileSize;
+
+    public FileAnalyzeResponse(String name, Long size, String type){
+        fileName = name;
+        fileSize =size;
+        fileType = type;
+    }
+
+    public String getFileName() {
```

Comment:
Its been a while we do this in Java, thanks to [Lombok](https://projectlombok.org/).

Just mentioning in case you don't see it in any Spay projects or any other Java project.

Reply:
I know I just want to feel myself small, I will get on with lombok to get rid of these boiler plates